README.md

```diff
@@ -61,6 +61,12 @@ operations on the wallets. The package has excellent documentation for a smooth
 
 ***starknet simpleCall***
 
+1. Rename the ".env.template" file located at the root of the "examples" folder to ".env"
+1. Uncomment, and assign your Sepolia testnet endpoint to the `RPC_PROVIDER_URL` variable in the ".env" file
+1. Uncomment, and assign your account address to the `ACCOUNT_ADDRESS` variable in the ".env" file
+1. Make sure you are in the "simpleCall" directory
+1. Execute `go run main.go`
+   
 ```sh
 cd examples/simpleCall
 go mod tidy
```

Comment:
If you pay attention, this line is no longer needed in all examples. Remove it from each quote

Reply:
I have removed those steps and ``go mod tidy``